lib/susedistribution.pm

```diff
@@ -644,6 +644,14 @@ sub activate_console {
             # case the system is still booting (https://bugzilla.novell.com/show_bug.cgi?id=895602)
             # or when using remote consoles which can take some seconds, e.g.
             # just after ssh login
+            my $retry_num = 0;
+            # retry send ctrl-alt-f key since system not response sometimes
+            while (!(check_screen \@tags, 60) && ($retry_num < 3) && !(check_var('VIRSH_VMM_FAMILY', 'hyperv'))) {
+                $retry_num++;
+                record_soft_failure("system not response refer to poo#45575,try send key again");
+                my $key = "ctrl-alt-f" . $nr;
```

Comment:
We already define this hotkey in os-autoinst. Now we have it here again. This is not so nice.

Reply:
To be fair, this would belong in the distribution anyway, but removing it from os-autoinst will cause too much confusion.

Reply:
My suggestion keep it here so you can easy get idea which key sent it. Reopen it if you not agree:)

Reply:
Sorry, I still do not think it's a good idea to repeat that definition anywhere.

Reply:
After further investigation, the hotkey defined in trigger_select func of ttyConsole.pm belong to backend process, so $self->screen->{console_key}  this kind of definition only can be seen within backend not in distribution.